### PR TITLE
[IMP] l10n_se: Adding SRU codes to account table.

### PR DESCRIPTION
**Pull request by daniel-stenlov**
Adding SRU codes to l10n_se for income tax reports and SIE import/export.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_se/data/account_tag.xml [hunk 1]
```diff
@@ -0,0 +1,431 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (jco-odoo) [inline addons/l10n_se/data/account_tag.xml:2]: xml because we want to set noupdate?

**Comment by jco-odoo** [comment]
@larmar If you would like to add P&L and balance sheet, please let us know.  @tsb-odoo

File: addons/l10n_se/data/account_tag.xml (hunk 1 above)
Comment (daniel-stenlov) [reply addons/l10n_se/data/account_tag.xml:2]: Do you want it as CSV?

Comment (jco-odoo) [reply addons/l10n_se/data/account_tag.xml:2]: That was indeed my point.

Comment (daniel-stenlov) [reply addons/l10n_se/data/account_tag.xml]: PR, updated and changed into CSV.

**Comment by larmar** [comment]
Hello Josse,

Yes I have P&L and Balance sheet. The P&L and balance sheet is in
l10n_se_reports ?

**Comment by jco-odoo** [comment]
@larmar Yes, we put the balance sheet and P&L report in the _reports module in enterprise.  Currently, the l10n_se_reports module has only an extension for the tax report to export the XML.  But as it is probably based on tags, we need to add those tags as well.  So, I would need an adaptation here and an adaptation of the l10n_se_reports module.

**Comment by larmar** [comment]
There is no need to add tags. While updating the reports working on l10n_se and updating the tags. I realized i did not need them.
Its only in _reports we need add code.

**Pull request closed without merging**